### DISCUSSION TITLE
quick_reply Object

### DISCUSSION THREAD

**Discussion by neil7227** [1101-GS3073-C/Python-Community-GS3073-C#313] [Q&A]
上次作業裡的LocationSendMessage函數裡的quick_reply是做甚麼的。
(class) LocationSendMessage(title=None, address=None, latitude=None, longitude=None, quick_reply=None, **kwargs: Any)

quick_reply object又是甚麼形式
:param quick_reply: QuickReply object
:type quick_reply: T <= linebot.models.send_messages.QuickReply

**Comment by 32844583** [comment]
![20121176a4rCOkQkPk](https://user-images.githubusercontent.com/66071134/208687595-c41bf885-1468-483d-a8d1-b6db5ac213ab.png)
在LINE回傳文字訊息的同時會跳出幾個按鈕，讓使用者可以更快速的進行下一步選擇
更多詳情可以參考以下文章
https://ithelp.ithome.com.tw/articles/10242373

**Comment by NcuJerryKao** [comment]
感覺去年的作業好難喔
